frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Link from "../Link";
+
+export interface BreadcrumbsProps {
+  links: { title: string; href: string }[];
+}
+
+const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => {
+  // taking in the array of links as defined above, create an array of Link components
+  const arrayLinks = links.map((link, index) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <Link href={link.href} title={link.title} key={index}>
+      {link.title}
+    </Link>
```

Comment:
So if href is optional you can do something like this:
```suggestion
    link.href ? (
      <Link href={link.href} title={link.title} key={index}>
        {link.title}
      </Link>
    ) : (
      <p key={index}>{link.title}</p>
    )
```
(I'm not sure if <p> will look right but you get my drift)

Reply:
ooh, neat, thanks
yeah, `<span>` is what looks right as in fits in line with the element